demo/index.html

```diff
@@ -544,48 +562,66 @@ <h5 class="u-text--right">Example</h5>
+                                <p class="docs-text--avoid">Avoid going lower than 3 column units as content will get too small and not flow correctly.</p>
                             </div>
-                            <p class="docs-text--avoid col-8">Avoid going lower than 3 column units as content will get too small and not flow correctly.</p>
                         </div>
                         <div class="col-12">
                             <h5 class="u-text--right">Example</h5>
                             <div class="docs-example">
                                 <div class="docs-example__item">
-                                    <div class="row">
-                                        <div class="col-12 box">.col-12</div>
-                                    </div>
-                                    <div class="row">
+                                    <div class="col-12 u-margin--bottom box">.col-12</div>
+                                    <div class="col-12 u-margin--bottom">
                                         <div class="col-11 box">.col-11</div>
                                         <div class="col-1 box">&nbsp;</div>
                                     </div>
-                                    <div class="col-10 box">.col-10</div>
-                                    <div class="col-2 box">.col-2</div>
-                                    <div class="col-9 box">.col-9</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-8 box">.col-8</div>
-                                    <div class="col-4 box">.col-4</div>
-                                    <div class="col-7 box">.col-7</div>
-                                    <div class="col-5 box">.col-5</div>
-                                    <div class="col-6 box">.col-6</div>
-                                    <div class="col-6 box">.col-6</div>
-                                    <div class="col-4 box">.col-4</div>
-                                    <div class="col-4 box">.col-4</div>
-                                    <div class="col-4 box">.col-4</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-2 box">.col-2</div>
-                                    <div class="col-8 box">.col-8</div>
-                                    <div class="col-2 box">.col-2</div>
-                                    <div class="col-3 box">.col-3</div>
-                                    <div class="col-6 box">.col-6</div>
-                                    <div class="col-3 box">.col-3</div>
+                                    <div class="col-12 u-margin--bottom">
```

Comment:
@anthonydillon Row in MAAS has padding set so col-12 is the better way to state a 100% width block